### PR TITLE
Update fine-grain.adoc

### PR DESCRIPTION
**Pull request by PoojaChandak**
Observed a few changes, suggesting the same. Kindly check.

### CODE REVIEW

**Comment by PoojaChandak** [comment]
Hi, these are minor typo/grammatical change suggestions. Kindly check and let me know if any updates required.

File: server_admin/topics/admin-console-permissions/fine-grain.adoc [hunk 1]
```diff
@@ -110,9 +110,9 @@ This admin is now able to manage this one client.
 
 ==== Restrict User Role Mapping
 
-Another thing you might want to do is to restrict the set a roles an admin is allowed
+Another thing you might want to do is to restrict the set roles an admin is allowed
```
Comment (stianst) [inline server_admin/topics/admin-console-permissions/fine-grain.adoc]:
```suggestion
Another thing you might want to do is to restrict the set of roles an admin is allowed
```

**Review by stianst — CHANGES_REQUESTED** [review]
Thanks, one minor thing to change then this is good to go

File: server_admin/topics/admin-console-permissions/fine-grain.adoc (hunk 1 above)
Comment (PoojaChandak) [reply server_admin/topics/admin-console-permissions/fine-grain.adoc]: Review comment update added. Kindly check.

**Review by stianst — APPROVED** [review]

**Pull request merged**